content/en/docs/refguide/modeling/application-logic/workflows/boundary-events.md

```diff
@@ -0,0 +1,105 @@
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/call-microflow.png" alt="Call Microflow" class="no-border" >}} | [Call microflow](/refguide/call-microflow/)
+| {{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-elements/call-workflow.png" alt="Call Workflow" class="no-border" >}} | [Call workflow](/refguide/call-workflow/)
+
+## Getting started
+
+To get started with the new boundary events feature, these can be enabled by opening Studio Pro's settings and navigating to new features (this setting is only applicable during the beta).
+
+- Windows: Preferences -> New Features -> Enable non-interrupting timer boundary events (beta)
+- Mac (Beta): Preferences -> New Features -> Enable non-interrupting timer boundary events (beta)
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/boundary-events-experimental-option.png" alt="Experimental option" width="450" class="no-border">}}
+
+## Adding Timer Boundary Events
+
+To add a boundary event to one of these activities there are 2 ways:
+
+1. Select an Event activity (currently only Timer event is available) from the workflow Toolbox and drag it on one of the activities listed above. Double click the Timer Boundary Event icon to open the Timer properties dialog.
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/drag-drop.png" alt="Adding boundary events through drag and drop" width="200" class="no-border">}}
+
+2. Open the activity (e.g. User Task) properties (1), go to the Events tab (2) and create a new boundary event (3). Select the Timer event (4).
+
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/workflow-how-to-boundary-events/edit-dialog.png" alt="Adding boundary events through edit dialog" width="450" class="no-border">}}
+
+The next step is to configure the boundary timer properties like Caption and timer Duration. This could be a fixed period (after the parent task is initiated) or a date+time can be defined in an expression. For more information see [Timer Event](/refguide/timer-event/).
+
+## Rearranging
+
+You can change the ordering of the boundary event in 2 ways:
+
+1. Via the context menu (right mouse click) of the timer event and select **Move boundary event left** or **Move boundary event right**, or by using the keyboard shortcuts [Keyboard Shortcuts](/refguide/keyboard-shortcuts/#editors-only).
+ 
+{{< figure src="/attachments/refguide/modeling/application-logic/workflows/boundary-events/arrange-in-editor.png" width="350px" alt="Timer Event arrange in editor" class="no-border" >}}
+
+2. Open the edit dialog of the parent activity and change the order of the boundary event paths by altering the list. Note that this does not change the order execution of the paths, as this is dependent on the expiration of the timer
```

Comment:
Maybe extract `Note that this does not change the order execution of the paths, as this is dependent on the expiration of the timer` as a separate line instead of being part of the 2nd bullet point.

Reply:
I'll put this in an info alert under the 2nd bullet point.

Reply:
I applied yiyun's suggestion